testbed/tests/widgets/properties.py

```diff
@@ -409,10 +407,21 @@ async def test_background_color_transparent(widget, probe):
     original = probe.background_color
     supports_alpha = getattr(probe, "background_supports_alpha", True)
 
+    # Set the background color to something different
+    widget.style.background_color = RED
+    await probe.redraw("Widget background color should be RED")
+    assert_color(probe.background_color, named_color(RED))
```

Comment:
What is this test for? We already have a background color test - why do we need to test a normal background color in the transparent background color test?

Reply:
I think, this is a debugging artifact, that I had put on for visual inspection of widgets with transparent as default background color. I'll remove it. Thanks.